azavea/district-builder-dtl-pa
# Add social media handles question to signup form

## Overview

Add social media handles question to signup form.

### Checklist


## Testing Instructions

 * Run migrations
 * Try to sign up or manage account when logged in and see this input on the form.

Closes [#163271449](https://www.pivotaltracker.com/story/show/163271449)
